src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -27,6 +27,27 @@
  */
 class OptionsResolver implements Options
 {
+    /** @var string[] The list of internal PHP is_* functions */
```

Comment:
No need for the `@var`, this is a const.
I'd suggest removing the comment altogether.